core/src/de/tobiasroeser/mill/vcs/version/VcsState.scala

```diff
@@ -37,6 +39,19 @@ case class VcsState(
     s"$versionPart$commitCountPart$revisionPart$dirtyPart"
   }
 
+  /**
+   * By default we strip the leading v if a user uses it.
+   * Ex. v2.3.2 -> 2.3.2
+   * @param tag the tag to process
+   * @return either the stripped tag or the tag verbatim
+   */
+  private def stripV(tag: String) =
```

Comment:
Can we make this one public and with explicit return type, so it would be easier to re-used the logic.

Reply:
Sure, done.